spec/chunk_spec.js

```diff
@@ -0,0 +1,39 @@
+import { chunk } from '../src/chunk'
+
+describe('chunk', ()=> {
+  it('returns array of elements in an array when chunk size is 1', () => {
```

Comment:
The language of these tests is a little confusing - `returns array of elements in an array`.  Maybe `returns a new array composed of arrays of elements when...`